functions/Backup-DbaServiceMasterKey.ps1

```diff
@@ -37,7 +37,7 @@ function Backup-DbaServiceMasterKey {
         Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
 
     .NOTES
-        Tags: Certificate, Database
+        Tags: CertBackup, Certificate, Backup
```

Comment:
Are the first tags going to be used for Categorization on the webpage? If so, would this one be Certificates? or Security?

Reply:
This command is categorized under a backup category:

![image](https://user-images.githubusercontent.com/11204251/147764073-51dec0a4-f97f-49ba-b153-0b680f143192.png)

Which I just noticed is not even in listed under the category section at the top of the page. I didn't go through all the commands to reorder the tags but ones I did touch tried to make the first tag match the category on the web page.

The section we have the cert commands in is called `Security and Certificates`, so I added just `Security` as the tag to those. If you'd like **all** to match up I can go through those but that will take me a few more days to complete.

Reply:
We also have other sections like `Export` that are not true "tags" being used in the commands, it is just `Find-DbaCommand Export-` to get those to list out.

Since we are doing the work now I think it would be worth while to list all commands with their associated tag, best we can at least. Sections like `Export` and `Find` I think should be dropped completely. That way if I'm looking at `Security and Certificates` I can see all commands that pertain to that topic/category, including the backup commands, export and find commands.

The biggest one of all is the Database, that tag is used everywhere...heavily.